### PR TITLE
CMOS-248: Add Alerts for Service Crashes

### PR DESCRIPTION
**Pull request by charlie-hayes**
These alerts are intended to be temporary until it is decided if/how an agent will be used in CMOS.

### CODE REVIEW

File: microlith/loki/alerting/couchbase/couchbase-rules.yaml [hunk 1]
```diff
@@ -94,3 +94,90 @@ groups:
...
+          health_check_id: CB90046
+          health_check_name: indexerCrash
+          cluster: '{{ $labels.cluster }}'
+          node: '{{ $labels.hostname }}'
+        annotations:
+          title: Index Service Crash ({{ $labels.hostname }})
+          description: The Index Service on {{ $labels.hostname }} (cluster {{ $labels.cluster }}) has crashed.
+          remediation: Review indexer.log to identify the cause, or contact Couchbase Technical Support.
+
+# CB90047-queryCrash, currently couchbase-fluent-bit does not work with query.log
```
Comment (markspolakovs) [inline microlith/loki/alerting/couchbase/couchbase-rules.yaml:115]: K8S link, just for completeness?

**Review by markspolakovs — COMMENTED** [review]
One thing I need to check (more of a note to self than a review comment): how does the `label_format` play with the `hostname` label

File: docs/modules/ROOT/partials/cmos-health-checks.adoc [hunk 2]
```diff
@@ -60,6 +60,68 @@ However, if it is happening repeatedly or you do notice disruption in your clust
...
+*Condition*: A crash is seen in `eventing.log`.
+
+*Remediation*: Review `eventing.log` to identify the cause, or contact Couchbase Technical Support.
+
+*Further Reading*: https://docs.couchbase.com/server/current/learn/services-and-indexes/services/eventing-service.html[Eventing Service]
+
+[#CB90051]
+=== Analytics Crash (CB90051)
+
+*Background*: If the Analytics Service process (`cbas`) crashes, it will be restarted within a few seconds by the cluster manager.
```
Comment (markspolakovs) [inline docs/modules/ROOT/partials/cmos-health-checks.adoc:116]: Are you sure that `cbas` is the Java process? I'd imagine that to be something like `java` or `javaw`.

Comment (charlie-hayes) [reply docs/modules/ROOT/partials/cmos-health-checks.adoc:116]: Cant see any processes with java-y names. When I kill `cbas` though the JVM halts.

**Review by markspolakovs — APPROVED** [review]
Have you tested the LogQL? If so LGTM

**Comment by charlie-hayes** [comment]
Will do a final check then merge :)

**Pull request merged**